engineering_tools/carma

```diff
@@ -285,27 +343,60 @@ carma__fix_bag() {
 
     fi
 }
+
+__get_compose_env_files() {
+    # If variable is not set, modify to get default values from carma-config
+    RETURN_COMPOSE_ENV_FILES="/tmp/.env"
+
+    # If .env file exists in current directory, it takes higher precedence over carma-config .env file
+    if [ -f ".env" ]; then
+        echoerr "NOTE: .env file detected in current directory, considering its values for current carma session."
+        RETURN_COMPOSE_ENV_FILES="$RETURN_COMPOSE_ENV_FILES,.env"
```

Comment:
@adev4a I currently coded to consider current directory `.env` as long as it exists. This is the only  change from how docker works where it uses it if neither of COMPOSE_ENV_FILES nor `--env-file` is set. Since my finctionality uses COMPOSE_ENV_FILES, it is always technically set. But we still want the local `.env` to overwrite it, so I had to include it this way. lmk if it's confusing, but I think this is really minor and benign side-effect since carma users don't usually use COMPOSE_ENV_FILES files anyways.

Reply:
Yup, it makes sense to me to have the local .env file take precedence over the carma-config .env file. Thanks for including the comments in the script as well